pontoon/sync/formats/ftl.py

```diff
@@ -73,7 +78,7 @@ def get_comment(obj):
         for obj in self.structure.body:
             if type(obj) == ast.Message:
                 key = obj.id.name
-                translation = serializer.serialize_message(obj)
+                translation = serializer.serialize_entry(obj)
```

Comment:
As I mentioned in two other comments in this PR, you should prefer `serialize_entry` here.  It's meant to do exactly what you want. The fact that `serialize_message` is exposed can be considered a bug.